pkg/apis/marketplace/v1alpha1/operatorsource_types.go

```diff
@@ -20,6 +20,8 @@ type OperatorSourceList struct {
 	Items           []OperatorSource `json:"items"`
 }
 
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
```

Comment:
Did you actually mean to move this?

Reply:
Yes, I had to move it. Otherwise if I do `operator-sdk generate k8s` it removes `deepcopy` functions.